src/main/java/org/alfasoftware/soapstone/SoapstoneOpenApiReader.java

```diff
@@ -476,4 +460,27 @@ private RequestBody parametersToRequestBody(String operationId, Collection<Param
       return schema;
     }
   }
+
+
+  private Converter<?, ?> getParameterConverter(final XmlJavaTypeAdapters adaptersAnnotation, final Type type) {
```

Comment:
Should either name, document, or comment this so that it's clear we are checking the package and why (basically the comment you have added above can just come down here).